src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -22,7 +20,7 @@ public enum OperationKind
         VariableDeclarationGroup = 0x3,
```

Comment:
FWIW, I don't find the hex particularly useful in this class. It doesn't really matter what the enum bitpattern is, so the hex doesn't help me. Not something to change, just an observation.